x-pack/test/rule_registry/security_and_spaces/tests/basic/search_strategy.ts

```diff
@@ -0,0 +1,157 @@
+  getRuleForSignalTesting,
+  createRule,
+  waitForSignalsToBePresent,
+  waitForRuleSuccessOrStatus,
+} from '../../../../detection_engine_api_integration/utils';
+import { ID } from '../../../../detection_engine_api_integration/security_and_spaces/tests/generating_signals';
+import { QueryCreateSchema } from '../../../../../plugins/security_solution/common/detection_engine/schemas/request';
+
+// eslint-disable-next-line import/no-default-export
+export default ({ getService }: FtrProviderContext) => {
+  const esArchiver = getService('esArchiver');
+  const supertest = getService('supertest');
+  const bsearch = getService('bsearch');
+  const log = getService('log');
+
+  const SPACE1 = 'space1';
+
+  describe('ruleRegistrySearchStrategy', () => {
+    describe('logs', () => {
+      beforeEach(async () => {
+        await esArchiver.load('x-pack/test/functional/es_archives/observability/alerts');
+      });
+      afterEach(async () => {
+        await esArchiver.unload('x-pack/test/functional/es_archives/observability/alerts');
+      });
+      it('should return alerts from log rules', async () => {
+        const result = await bsearch.send<RuleRegistrySearchResponse>({
+          supertest,
+          options: {
+            featureIds: [AlertConsumers.LOGS],
+          },
+          strategy: 'ruleRegistrySearchStrategy',
+        });
+        expect(result.rawResponse.hits.total).to.eql(5);
+        const consumers = result.rawResponse.hits.hits.map(
+          (hit) => hit._source?.['kibana.alert.rule.consumer']
+        );
+        expect(consumers.every((consumer) => consumer === AlertConsumers.LOGS));
+      });
+    });
+
+    /**
+     * Seeing this error:
+     *    │ proc [kibana] [2022-02-22T08:53:59.624-05:00][INFO ][plugins.ruleRegistry] Installing namespace-level resources and creating concrete index for .alerts-security.alerts-default
+   │ info [r.suppressed] [node-01] path: /_index_template/_simulate_index/.internal.alerts-security.alerts-default-000001, params: {name=.internal.alerts-security.alerts-default-000001}
+   │      java.lang.IllegalStateException: Cannot remove non-existent alias [.siem-signals-default] for index [.internal.alerts-security.alerts-default-000001]
+      │ proc [kibana] [2022-02-22T08:53:59.680-05:00][ERROR][plugins.ruleRegistry] RuleDataWriterInitializationError: There has been a catastrophic error trying to install namespace level resources for the following registration context: security.
+   │ proc [kibana]     This may have been due to a non-additive change to the mappings, removal and type changes are not permitted. Full error: ResponseError: illegal_state_exception: [illegal_state_exception] Reason: Cannot remove non-existent alias [.siem-signals-default] for index [.internal.alerts-security.alerts-default-000001]
+   │ proc [kibana]     at prepareForWriting (/Users/chris/dev/repos/kibana/x-pack/plugins/rule_registry/server/rule_data_client/rule_data_client.ts:175:17)
```

Comment:
This appears to be a new bug in Elasticsearch where the simulate index template API throws an error if provided an index name that already exists and that existing index has an alias. Issue filed [here](https://github.com/elastic/elasticsearch/issues/84256).

The test below triggers the error because it creates 2 rules consecutively. The second rule execution finds that the alerts index already exists and attempts to ensure that the mappings on the existing index are up to date, as part of this process it simulates the existing index and triggers this error. We do have other tests in the security solution that will most likely run into this error as well, however, they were skipped during the most recent ES promotion.

Reply:
That's very helpful, thanks for the context!